config/config_test.go

```diff
@@ -0,0 +1,135 @@
+package config
```

Comment:
Please change `config` to `config_test`.
All test files should have `_test` postfix in the package name.